app/uk/gov/hmrc/fhregistrationfrontend/views/helpers/Helpers.scala

```diff
@@ -54,6 +55,33 @@ object Helpers {
     s"${address.addressLine1}$optionalLines<br>${address.countryCode}"
   }
 
+  def formatBusinessRegistrationAddress(address: BusinessRegistrationAddress): String = {
+    val addressLine1 = address.line1.trim
```

Comment:
If you use List and join cleverly, you can do this a lot neater.

See formatAddress above